syntax = "proto2";
package person;
import "protos/move.proto";

message Person {
  optional string id = 1;
  optional string color = 2;
  optional Position position = 3;
  repeated move.Move moves = 4;
}

message Position {
  optional int32 x_offset = 1;
  optional int32 y_offset = 2;
}
